### ISSUE TITLE
@link resolution questions and possible bug (tsdoc reference resolutions)

### ISSUE THREAD

**Issue by bradennapier** [microsoft/tsdoc#342]
So this is actually `TypeDoc` that is producing the issue, but as it says it is using the standard set for resolving links from tsdoc itself (and because they haven't responded to the issue itself), I figured I'd post it here in case it is a tsdoc issue!

If it is an issue and it would save significant time to submit a PR I will if it actually is reviewed but with typedoc not responding I didn't want to waste my time for something that might be ignored :-) 

If you guys could please review the detailed issue https://github.com/TypeStrong/typedoc/issues/2141 I would appreciate it.  I confirmed its an issue that isnt vscode or typescript compiler itself as a typescript team member indicated it is a bug in either tsdoc or typedoc and not either of those (they validate as expected by the typescript team member)

- I am guessing this is actually typedoc since I (think) the compiler adhered to typedoc for handling its doc links standard?  
- Original Issue on TypeDoc Repo: https://github.com/TypeStrong/typedoc/issues/2141
- Basically if you reference an imported value using @link it will not resolve unless you import directly (rather than import * as)

```ts
import * as enums from '../enums'

/** {@link enums.MyEnum} or {@link enums.MyEnum MyEnum} does not resolve on docs but does on vscode */

/** {@link MyEnum} does not resolve on vscode (it uses global reference lookup in docs so it resolves in docs) */
```

- so you have to choose if you want links to work in docs OR when using the code in vscode :(
